src/main/java/ch/uzh/ifi/hase/soprafs22/game/logger/GameLogger.java

```diff
@@ -0,0 +1,25 @@
+package ch.uzh.ifi.hase.soprafs22.game.logger;
+
+/**
+ * A Logger for {@link ch.uzh.ifi.hase.soprafs22.game.Game}.
+ */
+public interface GameLogger {
```

Comment:
can you create a package and namespace "interfaces" inside of logger and move the interfaces there, please? :)

And can you rename the interfaces to a name with a prepending capital I, like IGameLogger, IGameStatistics?

Reply:
afaik, what I did is the standard for interfaces and implementations (see https://stackoverflow.com/questions/2814805/java-interfaces-implementation-naming-convention ). I changed it to match what you say for consistency with other code.